typegen/strats/stub.py

```diff
@@ -1,5 +1,117 @@
-from typegen.strats.gen import TypeHintGenerator
+import pathlib
+import libcst as cst
+from mypy import stubgen
+import constants
+from tracing.ptconfig import load_config
+from typegen.strats.inline import InlineGenerator
 
 
-class StubFileGenerator(TypeHintGenerator):
-    ident = "stub"
\ No newline at end of file
+def _get_temp_folder_path(root: pathlib.Path) -> pathlib.Path:
```

Comment:
This sort of functionality should be covered by the builtin `tempfile` module, which is very nice to use

Reply:
Done